src/status_im2/contexts/quo_preview/reactions/react.cljs

```diff
@@ -27,11 +27,14 @@
       [preview/preview-container
        {:state      state
         :descriptor descriptor}
-       [rn/view {:padding-bottom 150}
+       [rn/view
+        {:padding-bottom 150
+         :align-items    :center}
         [rn/view
-         {:padding-vertical 60
-          :justify-content  :center
+         {:width            100
+          :padding-vertical 60
+          :justify-content  :space-evenly
           :flex-direction   :row
           :align-items      :center}
-         [quo2.reaction/reaction @state]
-         [quo2.reaction/add-reaction @state]]]])))
+         [quo/reaction @state]
```

Comment:
Same for these preview screens, we should only have one design system component per preview screen- could you fix this here or as a follow up? 🙏

Reply:
was thinking of the same. I think as a follow-up might be better.

Reply:
Sounds great, maybe we can some simple unit tests in that too and align some other best practices :)